pubsublite/internal/wire/subscriber.go

```diff
@@ -379,13 +379,17 @@ func (f *singlePartitionSubscriberFactory) New(partition int) *singlePartitionSu
 // multiPartitionSubscriber receives messages from a fixed set of topic
 // partitions.
 type multiPartitionSubscriber struct {
+	// Immutable after creation.
+	clients     apiClients
```

Comment:
I don't see where `clients.Close()` is called. Is this intentional?

Reply:
Ah my bad, I somehow failed to patch the clients.Close() call for the subscriber clients. Fixed.
Thanks for noticing!